### DISCUSSION TITLE
In the storm loading these days, indicate the problem of the small circle

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#71] [Q&A]
One day I found that due to the peak of the Chinese Spring Festival on the Internet, some CDN accelerated pirated video websites, stuck and interruptions started at first; there were three points of cycle flashes when interrupted. BUG, or pirated video website; what is the reason for the embedded playback plug -in? Seeking confusion-

**Comment by gz83** [comment]
> One day I found that due to the peak of the Chinese Spring Festival on the Internet, some CDN accelerated pirated video websites, stuck and interruptions started at first; there were three points of cycle flashes when interrupted. BUG, or pirated video website; what is the reason for the embedded playback plug -in? Seeking confusion-

@whxuezhe Please use incognito mode or use clean user data to open the relevant website for testing.

In addition, please make sure that there is no redundant software (rogue software, redundant anti-virus software, etc.) or viruses on the computer that interfere with the operation of the browser

**Reply by whxuezhe** [reply]
@gz83 AVAST is not redundant antivirus, is it? My computer has always been away from rogue software such as 360 and all kinds of useless computer butlers, this problem is one is the browser reason, I also encountered in Firefox, everything is normal in Google Canary, and the other is that it may pirate video websites, constantly changing and upgrading the embedded player framework code.

**Reply by gz83** [reply]
> @gz83 AVAST is not redundant antivirus, is it? My computer has always been away from rogue software such as 360 and all kinds of useless computer butlers, this problem is one is the browser reason, I also encountered in Firefox, everything is normal in Google Canary, and the other is that it may pirate video websites, constantly changing and upgrading the embedded player framework code.

@whxuezhe I don't define any anti-virus software as redundant software. Generally, one anti-virus software for personal computers is enough.

In addition, the problem you encountered requires multiple rounds of testing to locate the problem. You need to ensure that the test environment has no other interference as much as possible, and provide us with more information about the test.

**Reply by whxuezhe** [reply]
@gz83 Does this information help the problem solve? -The network RAX120 router, China Telecom Gigabit Family Network, Intel Killer Series Wireless Network Card.

**Reply by gz83** [reply]
> @gz83 Does this information help the problem solve? -The network RAX120 router, China Telecom Gigabit Family Network, Intel Killer Series Wireless Network Card.

@whxuezhe 

This information is not very helpful in solving the problem. You need to provide the following information, and organize and send it up. Please introduce markdown syntax to make the typesetting more tidy.

1. Your computer hardware configuration information

2. Problematic websites

3. Operation of different browsers
Firefox: version number, whether the problem is reproduced, whether the results are consistent in normal mode and in incognito mode

Chrome/Vallina Chromium: version number, whether the problem is reproduced, whether the results are consistent in normal mode and in incognito mode

Edge: version number, whether the problem is reproduced, whether the results are consistent in normal mode and in incognito mode

Please cooperate with the investigation of the issue, failure to do so will not help resolve the issue.

**Reply by whxuezhe** [reply]
@gz83 1.Your computer hardware configuration information
MSI GS77 i9 12900H Nuclear display of web videos generally called
RTX3080Ti laptop
Realtek Audio
DDR5 4800 32*2
WIFI6E Intle kill 1675i Wireless network card
Problematic websites
http://fcdm1.com
https://www.zbkk.net/zbktype/1.html

Operation of different browsers：chromium github packaging to understand the version of the code library and the Google Canary version
Firefox Version:Firefox Nightly111.0a1(alpha)
Is the same website

**Reply by gz83** [reply]
> @gz83 1.Your computer hardware configuration information MSI GS77 i9 12900H Nuclear display of web videos generally called RTX3080Ti laptop Realtek Audio DDR5 4800 32*2 WIFI6E Intle kill 1675i Wireless network card Problematic websites http://fcdm1.com https://www.zbkk.net/zbktype/1.html
> 
> Operation of different browsers：chromium github packaging to understand the version of the code library and the Google Canary version Firefox Version:Firefox Nightly111.0a1(alpha) Is the same website

@whxuezhe 

Operation of different browsers information is incomplete, please add, no further follow-up processing at this time

**Reply by whxuezhe** [reply]
@gz83 I forgot to tell you that Norton's Avast contains anti -tracking software; in some Google core browsers, except for some questionnaires, I do not erase the cookie; the other in the experimental characteristics is to open the COOKIE COOKIE. And I used the technical means to change the session online. Therefore, here I hope you can only consider the browser's re -positioning; thank you!

**Reply by gz83** [reply]
> @gz83 I forgot to tell you that Norton's Avast contains anti -tracking software; in some Google core browsers, except for some questionnaires, I do not erase the cookie; the other in the experimental characteristics is to open the COOKIE COOKIE. And I used the technical means to change the session online. Therefore, here I hope you can only consider the browser's re -positioning; thank you!

@whxuezhe Since you failed to provide more valid information, this discussion is closed
